File: font_end/fruitsapp_admin/src/Models/PriceProduct.js

class PriceProduct {
    constructor(id, product_id, price, currency, description, create_at, update_at) {
        this.id = id;
        this.product_id = product_id;
        this.price = price;
        this.currency = currency;
        this.description = description;
        this.create_at = create_at;
        this.update_at = update_at; 
    }
}

export default PriceProduct;

File: back_end/fruitsapp_backend/Models/PriceProduct.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace fruitsapp_backend.Models
{
	public class PriceProduct
	{
        [Key]
        public int Id { set; get; }
        public int product_id { set; get; }
        public decimal? price { set; get; }
        public string? currency { set; get; }
        public string? description { set; get; }
        public DateTime create_at { set; get; }
        public DateTime update_at { set; get; }
        public bool isDelete { set; get; }

        public Product? Product { set; get; }
    }
}

